server/plugins/CMakeLists.txt

```diff
@@ -66,7 +66,8 @@ endif()
 
 foreach(plugin ${plugin_sources})
 	string(REPLACE .cpp "" plugin_name ${plugin} )
-		add_library(${plugin_name} MODULE ${plugin})
+	add_library(${plugin_name} MODULE ${plugin})
+	target_link_libraries(${plugin_name} PRIVATE Boost::headers)
```

Comment:
I suppose this makes up for the deleted `boost_include_dirs` in its parent folder?

Reply:
Yes